spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +35,39 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      before do
+        @mock_request = instance_double("Request", :request,
+          uuid: nil, method: nil,
+          format: nil, remote_ip: nil,
+          filtered_parameters: nil,
+          filtered_env: { "foo" => "bar", "HTTP_bar" => "bar" }
+        )
+      end
+
+      it "can be initialized with one argument" do
```

Comment:
I don't think we need this spec - I'd be very surprised if we accidentally broke it, and it's covered by the instance method specs below anyway.